hazelcast/src/main/java/com/hazelcast/query/impl/IndexProvider.java

```diff
@@ -43,11 +43,13 @@
      * @return the created index instance.
      */
     InternalIndex createIndex(
-        IndexConfig config,
-        Extractors extractors,
-        InternalSerializationService ss,
-        IndexCopyBehavior copyBehavior,
-        PerIndexStats stats,
-        StoreAdapter storeAdapter
+            IndexConfig config,
+            Extractors extractors,
+            InternalSerializationService ss,
+            IndexCopyBehavior copyBehavior,
+            PerIndexStats stats,
+            StoreAdapter storeAdapter
     );
+
+    PerIndexStats createPerIndexStats(boolean ordered, boolean usesCachedQueryableEntries);
```

Comment:
I cannot find usages of this method in either OS or ENT. Could you please point me to the code where it is used?

Reply:
A leftover code. Used it before but then rewrote it and forgot to remove this method. Fixed. Thanks!